src/Sarif/HashUtilities.cs

```diff
@@ -7,11 +7,35 @@
 
 namespace Microsoft.CodeAnalysis.Sarif
 {
+    public class HashData
+    {
+        string md5;
+        string sha1;
+        string sha256;
+
+        public HashData()
+        {
+            this.md5 = String.Empty;
+            this.sha1 = String.Empty;
+            this.sha256 = String.Empty;
+        }
```

Comment:
I don't think we need to supply a default constructor. We don't have a scenario where we'd need to construct a `HashData` without already having in hand the arguments to the other constructor.